engines/python/setup/djl_python/huggingface.py

```diff
@@ -481,6 +482,8 @@ def wrap_text_generation_pipeline(self, hf_pipeline):
         def wrapped_pipeline(inputs, *args, **kwargs):
             model = hf_pipeline.model
             tokenizer = hf_pipeline.tokenizer
+            if self.hf_configs.apply_chat_template:
+                inputs = tokenizer.apply_chat_template(inputs, tokenize=False)
```

Comment:
could we check if request is chat API style and apply chat template automatically?

Reply:
Done